addons/calendar/static/src/views/attendee_calendar/attendee_calendar_model.js

```diff
@@ -94,7 +94,7 @@ export class AttendeeCalendarModel extends CalendarModel {
                     }
                     // Records will share the same rawRecord.
                     const record = { ...event };
-                    const attendeeInfo = data.attendees.find(a => (
+                    const attendeeInfo = data.attendees.findLast(a => (
```

Comment:
Why is this change required?

Reply:
We should check this. Creating a new attendee for every update seems like an error